### ISSUE TITLE
Add term transverse terminal limb defect

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10328]
**Preferred term label:**
Transverse terminal limb defect

**Synonyms**
Transverse terminal limb reduction

**Definition (free text, please give PubMed ID)**
Terminal transverse limb deficiency is a congenital anomaly that appears as an “amputation” at different levels of the arm, e.g. humerus, proximal forearm, wrist or finger. The limb is missing the terminal (distal) segment(s), with preservation of all the segment(s) proximal to the missing segment.
PMID: 34263545

**Parent term (use hpo.jax.org/app)**
Abnormality of the upper limb HP:0002817

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adams-Oliver syndrome (OMIM100300)

**Your nano-attribution (ORCID)**
A. Zankl 0000-0001-8612-1062, @azankl
L. Wit 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
cannot this also occur in the lower limb? Do we need to adjust the label?

**Comment by lisannesw** [comment]
@pnrobinson That is correct indeed. My suggestion would be to change the label to: Transverse terminal upper limb defect. And to change the synonym to: Transverse terminal upper limb reduction.

**Comment by lisannesw** [comment]
@pnrobinson Based on above comments and coming across different levels of terminal transverse limb defect in the literature, we would suggest adding a new structure for all the different terminal transverse defects (see #10479). This issue is about the transverse terminal **upper** limb defect and the above suggestion has been cleaned up to be more fitting.  

**Preferred term label:**
Transverse terminal upper limb defect

**Synonyms**
Transverse terminal upper limb reduction

**Definition (free text, please give PubMed ID)**
A congenital anomaly that appears as an “amputation” at different levels of the arm, e.g. humerus, proximal forearm, wrist or finger. The limb is missing the terminal (distal) segment(s), with preservation of all the segment(s) proximal to the missing segment.
PMID: 34263545

**Parent term (use hpo.jax.org/app)**
Transverse terminal limb defect #10479 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adams-Oliver syndrome (OMIM100300)

**Your nano-attribution (ORCID)**
A. Zankl 0000-0001-8612-1062, @azankl
L. Wit 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 

Adding new term: 
- ID: HP:6000918
- label: Transverse terminal upper limb defect
- parents: [Abnormality of the upper limb (HP:0002817)](https://hpo.jax.org/app/browse/term/HP:0002817)
- pmid: PMID:34263545
- def: A congenital anomaly that appears as an amputation at different levels of the arm, e.g. humerus, proximal forearm, wrist or finger. The limb is missing the terminal (distal) segment(s), with preservation of all the segment(s) proximal to the missing segment.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10328
- synonyms: Transverse terminal upper limb reduction(NONE)

**Comment by pnrobinson** [comment]
done
